lib/tagnews/utils/geocode_list.py

```diff
@@ -0,0 +1,13 @@
+import geocoder
+import time
+
+def lat_longs_from_geo_strings(lst):
+    lats_lons = []
+    for addr_str in lst:
+        g = geocoder.google(addr_str)
+        if g.latlng is None:
+            time.sleep(.5)
+            return lat_longs_from_geo_strings([addr_str])
```

Comment:
@jherzberg I don't think we want to `return` here, I think we want to do
```python
lat_lons.append(lat_longs_from_geo_strings([addr_str]))
```
right?

Reply:
Or actually, we want to do `lat_lons.extend(...)` because we'll get a list out of the function.